spot-vaults/contracts/BillBroker.sol

```diff
@@ -657,7 +641,12 @@ contract BillBroker is
                     x2: arSoftBound.lower,
                     y2: swapFeePercs.lower
                 }),
-                Line({ x1: 0, y1: swapFeePercs.lower, x2: ONE, y2: swapFeePercs.lower }),
+                Line({
+                    x1: arSoftBound.lower,
+                    y1: swapFeePercs.lower,
+                    x2: ONE,
```

Comment:
Should this be an ~infinite value?

Reply:
I have a question about the Line inputs to the other _computeFeePerc call too, but maybe easiest to sync offline.